<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DalFileWindow</class>
 <widget class="QMainWindow" name="DalFileWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabGeneral">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="2">
         <widget class="QLabel" name="labelPrintLevelTip">
          <property name="text">
           <string>Overall print level used for the rest of the calculations.</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="checkBoxDirect">
          <property name="text">
           <string>.DIRECT</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelPrintLevel">
          <property name="text">
           <string>Print level:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spinBoxPrintLevel">
          <property name="maximum">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelJobType">
          <property name="text">
           <string>Job type:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxJobType"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="labelJobTypeTip"/>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="labelDirectTip">
          <property name="text">
           <string>Construct 2-electron integrals &quot;on-the-fly&quot; (do not write to disc).</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelWaveFunction">
          <property name="text">
           <string>Wave function:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBoxWaveFunction"/>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="labelWaveFunctionTip">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabOptimize">
       <attribute name="title">
        <string>Optimize</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="textEditDalFile"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
